pkg/apis/extensions/v1beta1/types.go

```diff
@@ -286,6 +286,12 @@ type DeploymentStatus struct {
 
 	// Total number of non-terminated pods targeted by this deployment that have the desired template spec.
 	UpdatedReplicas int32 `json:"updatedReplicas,omitempty"`
+
+	// Total number of pods targeted by this deployment that are ready.
+	ReadyPods int32 `json:"readyPods,omitempty"`
```

Comment:
Is it important to distinguish the case where these values have not yet been computed (i.e. "unknown")? If so, this should be a pointer type. I haven't decided one way or the other yet, just thought I'd bring it up for discussion.